packages/circuit-ui/components/PhoneNumberInput/PhoneNumberInput.module.css

```diff
@@ -11,6 +11,12 @@
   flex-grow: 1;
 }
 
+.flag {
+  top: var(--cui-spacings-byte);
+  left: var(--cui-spacings-byte);
+  display: flex;
+}
```

Comment:
These styles are no longer needed once you remove the inline styles from the Flag component wrapper.